Bottom: cf5bf1afd5864703dd6a49a86c3b667b9519f796
Top:    4758995328e901f0db5273b5faac96592df5db5c
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-02 21:32:33 -0500

Refresh of zarlink

---

diff --git a/package/kernel/linux/modules/sound.mk b/package/kernel/linux/modules/sound.mk
index 5d3d62a..b02c2ba 100644
--- a/package/kernel/linux/modules/sound.mk
+++ b/package/kernel/linux/modules/sound.mk
@@ -163,13 +163,12 @@ $(eval $(call KernelPackage,sound-cs5535audio))
 
 define KernelPackage/sound-soc-core
   TITLE:=SoC sound support
-  DEPENDS:=+kmod-regmap
+  DEPENDS:=+kmod-regmap +kmod-sound-core
   KCONFIG:= \
 	CONFIG_SND_SOC \
 	CONFIG_SND_SOC_DMAENGINE_PCM=y \
 	CONFIG_SND_SOC_ALL_CODECS=n
   FILES:=$(LINUX_DIR)/sound/soc/snd-soc-core.ko
-  DEPENDS:=+kmod-sound-core
   AUTOLOAD:=$(call AutoLoad,55, snd-soc-core)
   $(call AddDepends/sound)
 endef
